_component_design/accessibility.md

```diff
@@ -0,0 +1,219 @@
+
+It is important to reserve the prominent border at 2px for failed text input as an additional visual indicator for error state.
+
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+{% figure [class:"image bg-light border"] %}
+![caption:"Default state"]({{site.url}}/assets/images/overview/accessibility/accessibility-inputs-1.svg)
+{% endfigure %}
+
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+{% figure [class:"image bg-light border"] %}
+![caption:"Error state"]({{site.url}}/assets/images/overview/accessibility/accessibility-inputs-2.svg)
+{% endfigure %}
+
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+#### Focus for Input Field
+
+In this example, we provide a clear visual feedback that the focus has moved to the input field by changing the border color and adding a highlight to give the user clear indication of where they’re currently taking action.
+
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+{% figure [class:"image bg-light border"] %}
+![caption:"Default state"]({{site.url}}/assets/images/overview/accessibility/accessibility-inputs-3.svg)
+{% endfigure %}
+
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+{% figure [class:"image bg-light border"] %}
+![caption:"Error state"]({{site.url}}/assets/images/overview/accessibility/accessibility-inputs-4.svg)
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+<div class="hxRow">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+### Tools
+
+These best practices are not exhaustive, and they are not necessarily applicable to every situation. However, they do cover the major areas for concern when designing for visual accessibility. It is more important to take away the principles, so that you can integrate them into your design processes. Ultimately, improving accessibility benefits all users by providing greater ease of use to everyone because well-designed products are both more usable and more accessible.
+
+There are many tools that can help you with accessibility. Check them out to learn more:
+
+[WebAim’s color contrast checker](https://webaim.org/resources/contrastchecker/): Insert foreground and background colors to know if they pass accessibility guidelines.
+
+[Toggle Grayscale](https://goo.gl/aFn2h4): Plugin built by the Helix development to quickly view prototypes published to th web in grayscale within the Chrome browser.
+
+[I want to see like the color blind](https://chrome.google.com/webstore/detail/i-want-to-see-like-the-co/jebeedfnielkcjlcokhiobodkjjpbjia?hl=en-GB): Plugin built to apply color blindness filters to your web page right within Chrome.
```

Comment:
"I want to see like the color blind" - i'm sorry, what?

Plug-in* built to apply filters specific to color deficiencies. Usable only for Chrome.

Reply:
I updated the sentence. @etytaylor As Alex noted — The name of this plug-in is a little... less than optimal, shall we say. Is it worth keeping this plug-in? Could we perhaps use another one? or remove this altogether?